### PR TITLE
Fix small issue when cleanup tmpdir

### PR DESCRIPTION
**Pull request by MaXinjian**
Signed-off-by: Ma Xinjian <maxj.fnst@cn.fujitsu.com>

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Fix small issue when cleanup tmpdir 

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: pkg/volume/hostpath/host_path_test.go [hunk 1]
```diff
@@ -197,7 +197,7 @@ func TestProvisioner(t *testing.T) {
 		t.Errorf("Expected reclaim policy %+v but got %+v", v1.PersistentVolumeReclaimDelete, pv.Spec.PersistentVolumeReclaimPolicy)
 	}
 
-	os.RemoveAll(hostPathCreator.basePath)
+	os.RemoveAll(fmt.Sprintf("/tmp/%s", hostPathCreator.basePath))
```
Comment (jingxu97) [inline pkg/volume/hostpath/host_path_test.go:200]: without this change, does the test fail?

Comment (MaXinjian) [reply pkg/volume/hostpath/host_path_test.go:200]: With or without this change, test  all pass. But it actually doesn't cleanup tempfile without this change.

Comment (jingxu97) [reply pkg/volume/hostpath/host_path_test.go:200]: thanks!

**Comment by MaXinjian** [comment]
sorry about my misoperation. Nothing need to change

**Comment by dims** [comment]
@MaXinjian will you be picking this back up. looks like there are some comments from reviewers.

/retest

**Pull request closed without merging**